packages/website/ts/components/wallet/wrap_ether_item.tsx

```diff
@@ -152,16 +158,18 @@ export class WrapEtherItem extends React.Component<WrapEtherItemProps, WrapEther
     }
     private _renderIsEthConversionHappeningSpinner(): React.ReactElement<{}> {
         return this.state.isEthConversionHappening ? (
-            <div className="pl1" style={{ paddingTop: 10 }}>
+            <div className="pl3 pr2" style={styles.conversionSpinner}>
                 <i className="zmdi zmdi-spinner zmdi-hc-spin" />
             </div>
-        ) : null;
+        ) : (
+            <div style={styles.conversionSpinnerPlaceholder} />
```

Comment:
can we set the css property `visibility` to `hidden` here in order to get rid of the magic number?

Reply:
👍